libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMap.java

```diff
@@ -582,6 +582,8 @@ private void initializeNavigationSymbolManager(MapView mapView, MapboxMap mapbox
     mapboxMap.getStyle().addImage(MAPBOX_NAVIGATION_MARKER_NAME, markerBitmap);
     SymbolManager symbolManager = new SymbolManager(mapView, mapboxMap, mapboxMap.getStyle());
     navigationSymbolManager = new NavigationSymbolManager(symbolManager);
+    SymbolOnStyleLoadedListener onStyleLoadedListener = new SymbolOnStyleLoadedListener(mapboxMap, markerBitmap);
+    mapView.addOnDidFinishLoadingStyleListener(onStyleLoadedListener);
```

Comment:
Quick question @LukasPaczos I've noticed that we're not removing / releasing the listener from our side. @danesfeder mentioned that this wasn't needed. Is `MapView#onDestroy` releasing all the resources / making sure that any of the listeners don't leak? If so, is this something remarked in the docs? I only wanted to make sure that customers using the APIs don't mess up / the docs reflect how to properly use them. As a developer if I add a listener I'd normally try removing it afterwards based on the lifecycle implemented from "my" side so I think we should make this 💯 clear.

Reply:
> Is MapView#onDestroy releasing all the resources / making sure that any of the listeners don't leak?

Yes :)

Reply:
Awesome thanks for confirming! I believe that removing it explicitly isn't harmful, right?

Reply:
Shouldn't be harmful in any way.

Reply:
Yeah, makes sense! As mentioned, I think it's a good practice to remove all the resources previously added. But yeah thanks for confirming that we're backing up if any developers forget, so we don't crash 💪